s3tests_boto3/functional/test_s3.py

```diff
@@ -5102,6 +5102,38 @@ def test_bucket_recreate_not_overriding():
     objs_list = get_objects_list(bucket_name)
     eq(key_names, objs_list)
 
+
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='list all objects in a bucket using list_objects_v2')
+@attr(assertion='returns all expected objects')
+
+def test_listobjects_v2():
+    client = get_client()
+    bucket_names = []
+    for i in xrange(5):
```

Comment:
a little odd juggling names--but looks valid, must be similar logic for v1

Reply:
Yes, similiar logic is followed in v1